src/css/lib/yasr.css

```diff
@@ -1002,3 +1002,12 @@
   z-index: 11;
   width: auto;
   margin: inherit; }
+
```

Comment:
yasr.css contains the styles from the library and it's better to keep it as it is. The styles we extend in graphdb-workbench are in yars.custom.css